core/core.js

```diff
@@ -203,13 +203,13 @@ SharedQueue Binary Layout
     if ("ok" in res) {
       return res.ok;
     }
-    const ErrorClass = getErrorClass(res.err.className);
-    if (!ErrorClass) {
+    const errorBuilder = getErrorBuilder(res.err.className);
+    if (!errorBuilder) {
       throw new Error(
-        `Unregistered error class: "${res.err.className}"\n  ${res.err.message}\n  Classes of errors returned from ops should be registered via Deno.core.registerErrorClass().`,
+        `Unregistered error builder: "${res.err.className}"\n  ${res.err.message}\n  Builders for errors returned from ops should be registered via Deno.core.registerErrorBuilder().`,
       );
     }
-    throw new ErrorClass(res.err.message);
+    throw errorBuilder(res.err.message);
```

Comment:
This add an extra junk stack frame in op errors ☹ (#4189) Can we not keep error classes on the JS side?

Reply:
As described in the PR description: This is needed for WebGPU, where many errors
are `DOMException`s that take two parameters, the message and the error
type. Currently we have no way to create errors that take more than one
parameter.

Reply:
Can the infrastructure return a `[class, ...args]` tuple? It just needs to be constructed here.

Reply:
I guess that would work. I'll open another PR :upside_down_face: